packages/chain-agnostic-permission/src/index.test.ts

```diff
@@ -0,0 +1,16 @@
+import * as allExports from '.';
+
+describe('@metamask/chain-agnostic-permission', () => {
+  it('has expected JavaScript exports', () => {
+    expect(Object.keys(allExports)).toMatchInlineSnapshot(`
+      Array [
+        "caip25CaveatBuilder",
+        "Caip25CaveatType",
+        "createCaip25Caveat",
+        "Caip25EndowmentPermissionName",
+        "caip25EndowmentBuilder",
+        "Caip25CaveatMutators",
```

Comment:
quite a few more exports than just these i think

Reply:
Yes, unfortunately these tests don't cover type exports. I have some ideas floating around for how to solve that, but at the moment this is just JavaScript exports.

Reply:
ahh my bad. yeah that makes sense